packages/tjdashboard/administrator/views/dashboard/tmpl/default.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * @version    CVS: 1.0.0
+ * @package    Com_Tjdashboard
+ * @author     Techjoomla <extensions@techjoomla.com>
+ * @copyright  2017 Techjoomla
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+// No direct access
+defined('_JEXEC') or die;
+
+JHTML::_('behavior.modal');
+
+
+$document = JFactory::getDocument();
+JHtml::stylesheet(Juri::root().'media/techjoomla_strapper/bs3/css/bootstrap.css');
+
+$script  = 'var root_url = "' . Juri::root() . '";';
+$document->addScriptDeclaration($script, 'text/javascript');
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/morris/assets/js/raphael.min.js');
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/morris/assets/js/morris.min.js');
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/morris/assets/js/renderer.js');
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/tabulator/assets/js/jquery-ui.min.js');
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/tabulator/assets/js/tabulator.min.js');
+$document->addStylesheet('plugins/tjdashboardrenderer/tabulator/assets/css/tabulator_semantic-ui.min.css');
+$document->addStylesheet(Juri::root().'components/com_tjdashboard/assets/css/dashboard.css');
+$document->addStylesheet(Juri::root().'plugins/tjdashboardrenderer/countbox/assets/css/countbox.css');
+$document->addStylesheet(Juri::root().'media/techjoomla_strapper/css/bootstrap.j3.min.css');
+
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/tabulator/assets/js/renderer.js');
+$document->addScript(Juri::root().'components/com_tjdashboard/assets/js/tjDashboardService.js');
+$document->addScript(Juri::root().'components/com_tjdashboard/assets/js/tjDashboardUI.js');
+
+$document->addScript('https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js');
+$document->addScript(Juri::root().'plugins/tjdashboardrenderer/chartjs/assets/js/renderer.js');
```

Comment:
@monikaatvowel Check the requirement of this view.

Reply:
removed for now